server/vitals.js

```diff
@@ -161,7 +185,20 @@ async function checkButtonHeartbeat() {
 
             await db.updateButtonsSentVitalsAlerts(button.id, true)
 
-            // TODO Also send a text message to Responders and Heartbeat Phone Numbers once we know that these messages are reliable
+            // Check for inactive gateways - assume if gateway is sending vitals but sent_vitals_alert is null don't send client message
+            const gateways = await db.getInactiveGatewaysWithClient(client)
+            if (gateways.length > 0) {
+              // Store the client info
+              if (!clientMessages[client.id]) {
+                clientMessages[client.id] = {
+                  client,
+                  disconnectedButtons: [],
+                  reconnectedButtons: [],
+                }
+              }
```

Comment:
As we talked about on a call, since the `sent_vitals_alert_at` _is_ in fact being set to `NULL` when the gateway comes back online, this whole comment can be ignored!

Reply:
Thanks Theresa! :)